support-frontend/assets/helpers/urls/externalLinks.ts

```diff
@@ -12,13 +12,16 @@ import type { ReferrerAcquisitionData } from 'helpers/tracking/acquisitions';
 import { deriveSubsAcquisitionData } from 'helpers/tracking/acquisitions';
 import type { Option } from 'helpers/types/option';
 import { getBaseDomain, getOrigin } from 'helpers/urls/url';
+
 // ----- Types ----- //
-export type MemProduct = 'patrons' | 'events';
```

Comment:
Removed 'patrons' here to fix a typing issue but I believe this is also a remnant from when patrons was on the membership subdomain